### ISSUE TITLE
Dataset cannot run

### ISSUE THREAD

**Issue by JasmineDeng** [ai-tor/VPilot#25]
I'm unsure if this is a problem with DeepGTAV itself or how I setup the game, but while I can run the drive script just fine, the dataset script doesn't work. It works if I comment out the Dataset call, but if it is in the code, after I connect successfully to the DeepGTAV endpoint, I get core errors with the DeepGTAV.asi file or frame not found errors, and the game crashes. I'm using the most recent version of GTAV and the most recent version of DeepGTAV.

I'm unsure of what could be causing this error, seeing as drive works just fine, but the addition of the Dataset object causes it to fail.

**Comment by JasmineDeng** [comment]
To be more specific, I seem to be getting: "ERROR: Failed to receive frame."

**Comment by JasmineDeng** [comment]
Was a bug in my own setup.

**Comment by JasmineDeng** [comment]
Sorry, it seems like the bug reappeared. I cannot run if I pass in a Dataset object to the Start message. More specifically, it breaks if Dataset receives a "reward" argument.

**Comment by PurnaKoteswaraRaoMallepaddi** [comment]
Yes, i am also getting the same error for a  day now , but what i have found is the vesion of the game is not satisfying the deep gta v requirements . If you get any solution please answer .
